src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+.page {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  box-sizing: border-box;
+  color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used but not defined. Please define this variable to ensure the text color is applied correctly.